### ISSUE TITLE
ListView Command set visible on List and not on Document Library.

### ISSUE THREAD

**Issue by VesaJuvonen** [SharePoint/sp-dev-docs#2009]
_From @mubash7 on May 15, 2018 19:20_

#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior
For Listview Command set, if RegID is 101, the command set should be deployed in document library.

#### Observed Behavior
Command set visible in List and not on Document Library
[Elements.txt](https://github.com/SharePoint/sp-dev-docs/files/2011840/Elements.txt)
@VesaJuvonen 

#### Steps to Reproduce
Followed  instructions on MS tutorial:
https://docs.microsoft.com/en-us/sharepoint/dev/spfx/extensions/get-started/building-simple-cmdset-with-dialog-api

#### Submission Guidelines
I have read that changing the RegId fixes the issue but does not seems to be happening in my case.
https://github.com/SharePoint/sp-dev-docs/issues/1628

Thanks for your contribution! Sharing is caring.

_Copied from original issue: SharePoint/sp-dev-fx-extensions#133_

**Comment by VesaJuvonen** [comment]
_From @sanfo168 on June 1, 2018 18:35_

It could be that your ClientSideComponentID does not have an ID associated with it.  Try going into your UpRevisionCommandSet.manifest.json file and use the ID from there as the ClientSideComponentID in the elements.xml.

Hope this helps!

**Comment by nduchemin1** [comment]
I have the same problem in SP2019. The Command is only visble in lists. When i deploy the same package to SPOnline i see it in lists and document libraries.
Is there any news for this?

**Comment by donavanmarais** [comment]
Apologies for hijacking this closed thread, but after following these suggestions, I am still unable to get the boilerplate provisioned extension to work on a document library. Any help would be appreciated. Thanks

**Comment by shivdk7** [comment]
@andrewconnell @VesaJuvonen @donavanmarais  Even I'm facing the same issue, command set is not visible for  RegID 101 document library!!! I've tried the above mentioned approach but still issue persists,any further help would be appreciated.
Thanks.

**Comment by preetippatki** [comment]
Listview commandset was visible and suddenly disappeared. There is no error. While debugging the command is set to true but still does not show

**Comment by LastGunslinger** [comment]
Are there any new developments on this? I am also experiencing this issue, and have not been able to see my command set on either modern or classic document libraries. I have tried the steps suggested above, and have tried setting my RegistrationID to both 101 and 119 (suggested in issue #4136), but nothing seems to work.

**Comment by dbobrovsky** [comment]
I have this issue also. 
[test.zip](https://github.com/SharePoint/sp-dev-docs/files/5271175/test.zip)

Help me pls

**Comment by AJIXuMuK** [comment]
Hi @dbobrovsky @LastGunslinger @preetippatki - sorry for long delay. Could you please check if that's still the issue?
We are unable to reproduce it, seems to be fixed a while ago.
Thanks!

**Comment by LastGunslinger** [comment]
@AJIXuMuK I don't believe that this was fixed by a software update, I needed to fix my _elements.xml_ and _ClientSideInstance.xml_ configuration files. For reference, this is how the files are set up currently:

elements.xml
`
<?xml version="1.0" encoding="utf-8"?>
<Elements xmlns="http://schemas.microsoft.com/sharepoint/">
    <CustomAction
        Title="SpfxSpApproval"
        RegistrationId="101"
        RegistrationType="List"
        Location="ClientSideExtension.ListViewCommandSet"
        ClientSideComponentId="00000000-0000-0000-0000-000000000000"
        ClientSideComponentProperties="{&quot;sampleTextOne&quot;:&quot;One item is selected in the list.&quot;, &quot;sampleTextTwo&quot;:&quot;This command is always visible.&quot;}">
    </CustomAction>
    <CustomAction
        Title="SpfxSpApproval"
        RegistrationId="100"
        RegistrationType="List"
        Location="ClientSideExtension.ListViewCommandSet"
        ClientSideComponentId="00000000-0000-0000-0000-000000000000"
        ClientSideComponentProperties="{&quot;sampleTextOne&quot;:&quot;One item is selected in the list.&quot;, &quot;sampleTextTwo&quot;:&quot;This command is always visible.&quot;}">
    </CustomAction>
</Elements>
`

ClientSideInstance.xml
`
<?xml version="1.0" encoding="utf-8"?>
<Elements xmlns="http://schemas.microsoft.com/sharepoint/">
    <ClientSideComponentInstance
        Title="SpfxSpApproval"
        Location="ClientSideExtension.ListViewCommandSet"
        ListTemplateId="101"
        Properties="{&quot;sampleTextOne&quot;:&quot;One item is selected in the list.&quot;, &quot;sampleTextTwo&quot;:&quot;This command is always visible.&quot;}"
        ComponentId="00000000-0000-0000-0000-000000000000" />
</Elements>
`
